src/main/java/numble/carrot/domain/user/service/UserService.java

```diff
@@ -0,0 +1,49 @@
+package numble.carrot.domain.user.service;
+
+import lombok.RequiredArgsConstructor;
+import numble.carrot.domain.user.dto.User;
+import numble.carrot.domain.user.dto.UserJoinForm;
+import numble.carrot.domain.user.dto.UserLoginForm;
+import numble.carrot.domain.user.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public Long join(UserJoinForm userJoinForm) {
+        if(userRepository.findByEmail(userJoinForm.getEmail()).isPresent()) {
+            throw new RuntimeException("이미 존재하는 유저 입니다.");
+        }
+        User user = userJoinForm.toEntity();
+        return userRepository.save(user).getId();
+    }
+
+    public User findMember(String email) {
+        return userRepository.findByEmail(email).orElseThrow(()->{
+            try {
+                throw new ClassNotFoundException();
+            } catch (ClassNotFoundException e) {
+                e.printStackTrace();
+            }
+            return null;
+        });
```

Comment:
`orElseThrow`가 아닌 `orElse`를 사용해서 null을 return하는 것은 어떨까요?

```suggestion
return userRepository.findByEmail(email).orElse(null);
```